src/attack.c

```diff
@@ -511,10 +558,7 @@ attack_host (struct scan_globals *globals, struct in6_addr *ip, GSList *vhosts,
                     "<description/><type/><name/></source></detail></host>",
                     ip_str);
 #if (PROGRESS_BAR_STYLE == 1)
```

Comment:
I think we can remove this preprocessor directive. It is always set to 1 and other modules does not hanlde other option than this for dead hosts.. The DEFINE is above in the same file.